app/ts/Elicitation/Interface/IPreference.ts

```diff
@@ -1,4 +1,9 @@
-export type ElicitationMethod = 'ranking' | 'precise' | 'matching' | 'choice';
+export type ElicitationMethod =
```

Comment:
Lead types with a T like we do the I for interfaces. So: TElicitationMethod 
Makes it more clear what it is, but also makes collisions with types from other packages less likely.